app/routes/projects/index.tsx

```diff
@@ -465,7 +465,7 @@ export default function Projects() {
                     aria-controls="panel2a-controls"
                     id="panel2a-header"
                   >
-                    <strong>Missing Roles</strong>
+                    <strong>Does not have Roles</strong>
```

Comment:
yeah Needed roles it's more descriptive

Reply:
But this is not `Needed role`, this filter is to get projects that don't have the selected role. For the `Needed role`, we have the `Looking for` field, which is a different field, and the filter is named just like that. So we have the following 3 filters:

1. `Looking for`: This is meant so that projects specify which roles they are looking for help. Saved as the `disciplines` field on the Project schema.
2. `Has Roles`: What projects have members with the selected roles?
3. `Does not have Roles`: Looks for projects that don't have the selected roles among all their members.